_layouts/test.html

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+bodyClass: tests
+---
+{% include apiSidebar.html %}
+<div class="documentation">
+    <div class="wrap">
+        <div class="rendered">
+            <article>
+                {{ content }}
+            </article>
+        </div>
+        <div id="mocha"></div>
+    </div>
+</div>
+<script src="{{ site.github.url }}/js/mocha.js"></script>
+<script>
+  mocha.setup('bdd');
+</script>
+<script src="https://cdn.rawgit.com/chaijs/chai/{{ site.data.releases[0].tag_name }}/chai.js"></script>
```

Comment:
Based on the comment above, this line should be removed also. Just leaving a comment here to remind you.